qsimcirq/qsim_simulator.py

```diff
@@ -50,7 +51,7 @@ def __init__(self,
                        final_simulator_state=final_simulator_state)
 
 
-class QSimSimulator(SimulatesAmplitudes, SimulatesFinalState):
+class QSimSimulator(SimulatesSamples, SimulatesAmplitudes, SimulatesFinalState):
```

Comment:
In order to keep the scope of this PR focused, please open a separate PR for this change.

Reply:
If you're happy with the state of the PR at commit 6a3a0bb, I'll merge it from there.

Reply:
Right, sorry I didn't intend for my new commit about sampling to be part of this PR. Happy to contribute up to the commit at 6a3a0bb

Reply:
I'm not sure I have the permissions needed to modify a branch in your fork of the repo - could you remove the most recent commit?

Reply:
Sorry for the delay. I have removed the most recent commit pertaining to Sampling. The files changed in this PR should now only be about the new randomized test. Thank you.